app/assets/stylesheets/application.scss

```diff
@@ -3,6 +3,11 @@
 $govuk-use-legacy-palette: true;
 $govuk-typography-use-rem: false;
 
+// govuk_frontend_toolkit dependencies
+@import "typography";
+@import "colours";
+@import "grid_layout";
+
```

Comment:
I'm confused by this commit - it doesn't include the bump to v19 as stated. Could it be merged into another commit?

Reply:
Damnit... that's down to the rebase from master I had to do this morning. Will fix.

Reply:
Fixed with a squash, rebase confused things.